app/assets/javascripts/related_tag.js

```diff
@@ -12,10 +12,9 @@
 
   Danbooru.RelatedTag.initialize_buttons = function() {
     this.common_bind("#related-tags-button", "");
-    this.common_bind("#related-general-button", "general");
-    this.common_bind("#related-artists-button", "artist");
-    this.common_bind("#related-characters-button", "character");
-    this.common_bind("#related-copyrights-button", "copyright");
+    $.each(JSON.parse(Danbooru.meta("related-tag-button-list")), function(i,category) {
+      Danbooru.RelatedTag.common_bind(`#related-${category}-button`, category);
```

Comment:
[Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) (`\`#related-${category}-button\``) are a ES6 feature and aren't supported in IE11. This will cause a Javascript parse error in that browser.